python/ql/src/meta/alerts/RemoteFlowSourcesReach.ql

```diff
@@ -0,0 +1,47 @@
+/**
+ * @name Remote flow sources reach
+ * @description Nodes that can be reached with taint tracking from sources of
+ *   remote user input.
+ * @kind problem
+ * @problem.severity recommendation
+ * @id py/meta/alerts/remote-flow-sources-reach
+ * @tags meta
+ * @precision very-low
+ */
+
+private import python
+private import semmle.python.dataflow.new.DataFlow
+private import semmle.python.dataflow.new.TaintTracking
+private import semmle.python.dataflow.new.RemoteFlowSources
+private import meta.MetaMetrics
+private import semmle.python.dataflow.new.internal.PrintNode
+
+class RemoteFlowSourceReach extends TaintTracking::Configuration {
+  RemoteFlowSourceReach() { this = "RemoteFlowSourceReach" }
+
+  override predicate isSource(DataFlow::Node node) {
+    node instanceof RemoteFlowSource and
+    not node.getLocation().getFile() instanceof IgnoredFile
+  }
+
+  override predicate isSink(DataFlow::Node node) {
+    not node.getLocation().getFile() instanceof IgnoredFile and
+    (
+      node instanceof RemoteFlowSource
+      or
+      this.isAdditionalFlowStep(_, node)
+    ) and
+    // we used to do `obj -> obj.meth` and `obj.meth -> obj.meth()` in two separate
+    // steps, and now do them in one `obj -> obj.meth()`. So we're going to ignore the
+    // fact that we no longer taint the node in the middle.
```

Comment:
```suggestion
    // In September 2021 we switched to a new taint tracking library.
    // To be able to compare the old taint tracking approach to the new one, we have to
    // consider the fact that the old taint tracking would create two taint steps for method
    // calls, `obj -> obj.meth` and `obj.meth -> obj.meth()`, while the new analysis creates
    // just a single step, `obj -> obj.meth()`. We handle this by excluding the node in the
    // middle.
    //
    // We should be able to remove this code once we don't care to compare the two
    // libraries anymore.
```

Reply:
I moved your suggestion to the relevant conversation, so closing this one